### PR TITLE
doc: document mibserver troubleshooting and update CHANGELOG

### PR DESCRIPTION
**Pull request by omrozowicz-splunk**
# Description

This PR:
1. Documents how to verify if the mibserver compiled files correctly
2. Updates the CHANGELOG

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Refactor/improvement

## How Has This Been Tested?

N/A

## Checklist

N/A

### CODE REVIEW

**Review by harshit-splunk — APPROVED** [review]

File: docs/mib-request.md [hunk 1]
```diff
@@ -70,12 +70,18 @@ localMibs:
   pathToMibs: "/home/user/local_mibs"
 ```
 
-It creates a Kubernetes pvc with MIB files inside and maps it to MIB server pod.
-Also, you can change the storageClass and size of persistence according to the `mibserver` schema: [check here](https://github.com/pysnmp/mibs/blob/develop/charts/mibserver/values.yaml).
+To verify if the process of compilation was completed successfully, check the mibserver logs with:
 
-4. Whenever you add new MIB files, rollout restart MIB server pods to compile them again:
+```bash
+microk8s kubectl deployments/snmp-mibserver
```
Comment (wojtekzyla) [inline docs/mib-request.md:76]: Shouldn't it be `microk8s kubectl logs deployments/snmp-mibserver` to get logs?

Comment (wojtekzyla) [inline docs/mib-request.md:76]: Shouldn't it be `microk8s kubectl logs deployments/snmp-mibserver` to get logs?

Comment (omrozowicz-splunk) [reply docs/mib-request.md]: Right, fixed!

**Review by wojtekzyla — APPROVED** [review]

**Pull request merged**